langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -292,6 +413,14 @@ public isolated function strip(xml x) returns xml = @java:Method {
 # If parameter `nm` is `()`, selects all elements;
 # otherwise, selects only elements whose expanded name is parameter `nm`.
 #
+# ```ballerina
+# xml codingBooks = xml `<!--Web dev--><script>JS</script><!--API development--><code>Ballerina</code>
```

Comment:
Please revisit variable names. Can just be `books`.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/8bceaabd7a1a2202d2929fee34d1e15e2f2b9b18